src/client/pages/TripsPage/TripsPage.jsx

```diff
@@ -96,6 +104,32 @@ class TripsPage extends Component {
       return null;
     }
   }
+
+  // Helper function for modal
+
+  showModal(tripId) {
+    this.setState({ isModalShown: true });
+    this.setState({ modalTripId: tripId });
```

Comment:
this can be in one statement. Remember that `this.setState()` takes in an object as parameter.
so..

`this.setState({ isModalShown: true, modalTripId: tripId })`